app/views/cases/shared/_closed_case_details.html.slim

```diff
@@ -1,7 +1,8 @@
 tbody.response-details
   - if case_details.date_responded.present?
     tr.date-responded
-      th = t('common.case.date_responded')
+      / i18n-tasks-use t('common.case.date_responded')
+      th = translate_for_case(case_details, 'common.case', 'date_responded')
```

Comment:
This translation isn't doesn't change depending on the class of case ... why use `translate_for_case` here?